### ISSUE TITLE
Maintenance/Local login does not have labels for username and password

### ISSUE THREAD

**Issue by raphendyr** [apluslms/a-plus#582]
Currently, the form is missing label fields, but that information is currently delivered via placeholders. However, that is not enough.

Solutions ideas:

* just add labels there. maybe placeholders should be removed?
* implement something like material design, and move the label away when field is focused (see [Material Design / Text Fields](https://material.io/components/text-fields) and [Material Design for Bootstrap / labels](https://fezvrasta.github.io/bootstrap-material-design/docs/4.0/material-design/labels/))

Image of the current state (note the warnings about missing labels):

![screenshot of accessibility issues on login form](https://user-images.githubusercontent.com/43036333/84796419-465a4b80-b001-11ea-9f1f-c4142fdae759.png)

**Comment by etanttila** [comment]
@Mankro I understood that @mtthwhggns would be dealing with this as it concerns accessibility.

**Comment by Mankro** [comment]
@etanttila could add the labels to the login form. You already worked on the same login form, didn't you? I like the simple idea of adding normal labels to the form fields. We don't need any fancy "material design" tricks that Jaakko suggested. @mtthwhggns could briefly review the pull request (when it is done) from the accessibility point of view.

**Comment by etanttila** [comment]
All right, I did work on hardcoding the texts of this login. Matthew did bring up the missing labels earlier and we though I might do it, but it turned out it wasn't exactly clear should be done (differences in the visions between Matthew and Jaakko), so that's why earlier we decided to postpone this. If its clear what should be done (just add the labels), I can do this. Would this be a "do as soon as possible" task or "do when you get stuck on (or sick and tired of) the course points page and need something else to do for a moment" task?

**Comment by etanttila** [comment]
Should I leave the placeholders or remove them?
![image](https://user-images.githubusercontent.com/50318434/85261900-9871f580-b475-11ea-8991-5beec3ed3288.png)
![image](https://user-images.githubusercontent.com/50318434/85262187-1b934b80-b476-11ea-8c19-45aeac72aa05.png)

**Comment by mtthwhggns** [comment]
I would say don't repeat the information: remove them. 

Placeholders should help people understand what input is expected, but here, I think it's clear enough without any.
